### ISSUE TITLE
Error when using Nester

### ISSUE THREAD

**Issue by mundodisco8** [tapnair/NESTER#6]
Hi!

First, thanks for making this tool and sharing it with others.

I'm trying to use Nester to flatten out a cutlery tray I want to lasercut ([Cutlery Tray v2 v7.zip](https://github.com/tapnair/NESTER/files/787172/Cutlery.Tray.v2.v7.zip)), but I got an error the first time I run it:

![2017-02-20_1017](https://cloud.githubusercontent.com/assets/8402707/23121075/edd1735a-f756-11e6-9aaa-887c40f8e177.png)

I select all the components of my model to be laid out onto a board.

After that, the Nester button disappears from the UI, and I get this error if I try to run it again:

![2017-02-20_1018](https://cloud.githubusercontent.com/assets/8402707/23121097/030ae8be-f757-11e6-98d7-7c25f13d5cf2.png)

Does it ring any bell?

Thanks!

**Comment by chesnel** [comment]
You have the obligation to define the "select Direction (edge)" Otherwise this causes this error ;)

**Comment by mundodisco8** [comment]
Hi,

After your reply, I tried again, but the error was still present even after selecting the direction (unless you were referring to the second error, the Runtime error).

What I did was stop and think for a bit, and I finally got what "Face is from the root component". I created a new component for the stock, where I lay my other faces, and then it worked as expected. My problem was that my stock was a body in the root component!

Thanks for the help, anyway!
